src/main/java/core/basesyntax/service/impl/DataProcessorServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.DataProcessorService;
+import core.basesyntax.strategy.impl.FruitStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class DataProcessorServiceImpl implements DataProcessorService {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final FruitStrategyImpl fruitStrategy = new FruitStrategyImpl();

```

Comment:
```suggestion
    private static final FruitStrategy fruitStrategy = new FruitStrategy();

```
this is not an implementation